sqlserver/datadog_checks/sqlserver/const.py

```diff
@@ -173,13 +181,18 @@
 #   4 = Suspect, 5 = Emergency, 6 = Offline, 7 = Copying, 10 = Offline_Secondary
 # Is Sync with Backup enum:
 #   0 = False, 1 = True
-DATABASE_METRICS = [
+DATABASE_FILES_METRICS = [
```

Comment:
do you think there is a way we can define these queries such that it makes it obvious what set of tags applies to each metric?

Reply:
This is what I planned for next refactor PR. Each metric will be an instance of a Pydantic model, something like
```python
class SQLServerMetric(BaseModel):
    metric: str
    table: str
    column: str
    tags: Optional[Set[str]] = None

    def __init__(self, **kwargs):
        super().__init__(**kwargs)
        if self.tags is None:
            self.tags = set()
```

Reply:
oh interesting. sounds _pythonic_

Reply:
is this a _sarcasm_ 🤣 ?